doc/api/http2.md

```diff
@@ -136,6 +136,9 @@ listener does not expect any arguments.
 added: v8.4.0
 -->
 
+* {Http2Session}
```

Comment:
What does this line mean? If this is the first parameter of an event listener, we need to add a name to it, as the only unnamed types in docs are types of properties and returned values. If this is an event emitter, I do not think we formally state them in this way, as this is inferred from the section place; this can baffle doctools.

Reply:
Umm, so should I add a name to it or skip it?

Reply:
If this is not a listener parameter, but an event source, I think we can skip it as this event is already a subsection of `Http2Session` section.

Reply:
So we need to add a name then for consistency if I am not mistaken. It seems we normally have not unnamed parameters in docs.

Reply:
naming it `session`, in that case.